spec/ransack/helpers/form_helper_spec.rb

```diff
@@ -849,6 +849,11 @@ module Helpers
         before do
           Ransack.configure { |c| c.search_key = :example }
         end
+
+        after do
+          Ransack.configure { |c| c.search_key = :q }
```

Comment:
Specs are really unstable for me locally. To me it looks like the configuration options and their modifications leak across specs and result in intermittent failures depending on the order in which specs are run. This is a good example: unless we return the option value back to its default, it will affect other specs.

Do you guys see that behavior locally or am I missing something?

Reply:
It's been a while since I ran the test suite myself, but if it's being run in a random order, `rspec bisect` can help identify the tests that are causing other tests to fail.